hil/cli/network.py

```diff
@@ -0,0 +1,80 @@
+"""Commands related to networks are in this module"""
+import click
+import sys
+from hil.cli.client_setup import setup_http_client
+
+C = None
+
+
+@click.group()
+def network():
+    """Commands related to network"""
+    global C
+    C = setup_http_client()
```

Comment:
Maybe move this logic to cli.main (and centralize the C variable)?

Reply:
I initialize a client object in `client_setup.py` and then the modules import it. Does that make it better? 
Since I import those modules in `cli.py`, by making the modules import client object from `cli.py` I got worried that I may run into issues with circular imports. Wasn't sure about it so I took the safer route.

Reply:
Fair enough. You could do what we've done with `hil.flaskapp`, but I don't feel strongly.